UnicornRender/system/source/profiler/WindowProfiler.cpp

```diff
@@ -19,15 +19,15 @@ namespace system
 WindowProfiler::WindowProfiler(Manager& manager)
     : m_systemManager( manager )
 {
-    LOG_INFO("WindowProfiler created");
+    spdlog::get(utility::Logger::LoggerName())->info("WindowProfiler created");
```

Comment:
`std::shared_ptr<spdlog::logger>` should be a class variable. There is no need to call `get()` every time.